src/security/security-tests/src/test/java/org/geoserver/security/password/URLMasterPasswordProviderTest.java

```diff
@@ -46,4 +46,33 @@ public void testEncryption() throws Exception {
         char[] passwd = mpp.doGetMasterPassword();
         assertTrue(Arrays.equals("geoserver".toCharArray(), passwd));
     }
+
+    @Test
+    public void testSecuredAccess() throws Exception {
```

Comment:
This seems like an unrelated change, or not? Please elaborate.

Reply:
removed